tests/ui/redundant_closure_call_fixable.fixed

```diff
@@ -102,3 +102,12 @@ mod issue11707 {
 fn avoid_double_parens() {
     std::convert::identity(13_i32 + 36_i32).leading_zeros();
 }
+
+fn fp_11274() {
+    macro_rules! m {
+        ($closure:expr) => {
+            $closure(1)
+        };
+    }
+    m!(|x| println!("{x}"));
```

Comment:
Can we also have a test case for `|| ()` cases as in https://github.com/rust-lang/rust-clippy/pull/12082/files#diff-ba9477a2653088e57270e74982b211e736a8f70a75f320dcba6a3d6ee37f3f75R179

Reply:
If you mean something like:

https://github.com/rust-lang/rust-clippy/blob/6558956301bb314c14542e264fde83c10e1a5422/tests/ui/redundant_closure_call_fixable.rs#L16-L27
 
These are already checked earlier.

Reply:
Oh I missed that :D